integration/testdata/centos-7-ignore-unfixed.json.golden

```diff
@@ -4665,5 +4665,180 @@
         "LastModifiedDate": "2019-06-13T21:29:00Z"
       }
     ]
+  },
+  {
+    "Target": "",
+    "Class": "lang-pkgs",
+    "Type": "python-pkg",
+    "Vulnerabilities": [
+      {
+        "VulnerabilityID": "CVE-2016-5636",
+        "PkgName": "Python",
```

Comment:
I don't think they are vulnerable because this python was installed via OS package manager. We probably need to skip those files.